modules/base/rendering/screenspacerenderablerenderable.cpp

```diff
@@ -0,0 +1,306 @@
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF  *
+ * CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE  *
+ * OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                                         *
+ ****************************************************************************************/
+
+#include <modules/base/rendering/screenspacerenderablerenderable.h>
+
+#include <openspace/camera/camera.h>
+#include <openspace/rendering/renderable.h>
+#include <openspace/scene/rotation.h>
+#include <openspace/scene/scale.h>
+#include <openspace/scene/translation.h>
+#include <openspace/util/updatestructures.h>
+
+namespace {
+    constexpr openspace::properties::Property::PropertyInfo TimeInfo = {
+        "Time",
+        "Time",
+        "The time (in J2000 seconds) that is used to calculate transformations and the "
+        "renderable's data."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraPositionInfo = {
+        "CameraPosition",
+        "Camera Position",
+        "Specifies the location of the virtual camera that is showing the renderable "
+        "class. This position is provided in meters."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraCenterInfo = {
+        "CameraCenter",
+        "Camera Center",
+        "The location of the camera's focal point. The camera's view direction will "
+        "always be pointing at the provided center location. This position is provided "
+        "in meters."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraUpInfo = {
+        "CameraUp",
+        "Camera Up",
+        "The direction that is 'up' for the provided camera. This value does not have "
+        "any units."
+    };
+
+    constexpr openspace::properties::Property::PropertyInfo CameraFovInfo = {
+        "CameraFov",
+        "Camera Field of view",
+        "The camera's field of view in degrees."
+    };
+
+    const openspace::properties::PropertyOwner::PropertyOwnerInfo TransformInfo = {
+        "Transform",
+        "Transform",
+        "The Translation, Rotation, and Scale that are applied to the rendered "
+        "Renderable."
+    };
+
+    // This [ScreenSpaceRenderable](#core_screenspacerenderable) object can render any
+    // [Renderable](#renderable) type into an image that is shown in screen space. This
+    // can be used to display a rendered object as an overlay in front of the regular 3D
+    // rendering of the scene.
+    //
+    // Note to use this ScreenSpaceRenderable, it might be necessary to specify the `size`
+    // parameter, which determines the resolution of the inset window into which the
+    // Renderable is rendered. For many use cases, the default should suffice, however.
+    //
+    // A possible use-case for the ScreenSpaceRenderable would be to show a 3D model of a
```

Comment:
```suggestion
    // Note that to use this `ScreenSpaceRenderable`, it might be necessary to specify the `size`
    // parameter, which determines the resolution of the inset window into which the
    // Renderable is rendered. For many use cases, the default should suffice, however.
    //
    // A possible use-case for the `ScreenSpaceRenderable` would be to show a 3D model of a
```
?

The description looks great btw. Super clear :)